src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -206,15 +201,15 @@ private void TraceLogInvalidTx(ContractEntry entry, string reason)
         }
 
         /// <summary>
-        /// Executes the <paramref name="entry"/>.
+        ///     Executes the <paramref name="entry" />.
         /// </summary>
-        /// <param name="entry">Transaction entry to be executed inside the <see cref="VirtualMachine"/>.</param>
-        /// <param name="stateUpdate"><see cref="Delta"/> to be used for execution environment construction</param>
+        /// <param name="entry">Transaction entry to be executed inside the <see cref="VirtualMachine" />.</param>
+        /// <param name="stateUpdate"><see cref="Delta" /> to be used for execution environment construction</param>
         /// <param name="txTracer">Tracer to extract the execution steps for debugging or analytics.</param>
         /// <param name="readOnly">Defines whether the state should be reverted after the execution.</param>
         /// <exception cref="TransactionCollisionException">Thrown when deployment address already has some code.</exception>
         /// <exception cref="OutOfGasException">Thrown when not enough gas is available for deposit.</exception>
-        private void Execute(ContractEntry entry, StateUpdate stateUpdate, ITxTracer txTracer, bool readOnly)
+        private void Execute(PublicEntry entry, StateUpdate stateUpdate, ITxTracer txTracer, bool readOnly)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The Cyclomatic Complexity of this method is 34 which is greater than 10 authorized.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4941742)